.devcontainer/devcontainer.json

```diff
@@ -31,12 +41,6 @@
     "editor.formatOnSave": true,
     "editor.formatOnType": true,
     "files.trimTrailingWhitespace": true,
-    "terminal.integrated.profiles.linux": {
-      "zsh": {
-        "path": "/usr/bin/zsh"
-      }
-    },
```

Comment:
PS: this is not required. VS Code auto detects all shells and creates profiles to them.

Also, this was breaking my dotfiles from working inside of this devcontainer for some reason.